src/styles.scss

```diff
@@ -51,3 +51,22 @@ body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+.mat-snack-bar-container{
+  color: white;
+  font-weight: 500;
+  &.success{
+    background-color: #28a745!important;
```

Comment:
Seems like those `!important` are not needed. You should never use them unless it is not possible to style an element without them. After removing it from what I see element's background is still green :)